### PR TITLE
make alias purpose more clear

### PR DESCRIPTION
**Pull request by adhorrig**
cc @rosalanghammer 

screen shot 2016-04-21 at 17 17 56

### CODE REVIEW

File: web/public/templates/profiles/general-info.dust [hunk 1]
```diff
@@ -32,6 +32,9 @@
           <label for="alias" class="col-sm-4 control-label">{@i18n key="Alias"/}* </label>
           <div class="col-sm-8">
             <input type="text" name="alias" class="form-control" placeholder="{@i18n key="Alias"/}" ng-model="profile.alias">
+            <div class="cd-tooltip">
+              {@i18n key="This  is shown to other users when your child uses the youth forums or if they choose to make their profile public. It should be a <b>unique</b> nickname. Try adding numbers at the beginning or end to make your alias unique."/}
```
Comment (Wardormeur) [inline web/public/templates/profiles/general-info.dust]: Isn't the key wrong because of <b>unique</b> ?

**Comment by Wardormeur** [comment]
Conflict & rebase pleace :)

File: web/public/templates/profiles/general-info.dust [hunk 2]
```diff
@@ -32,6 +32,9 @@
           <label for="alias" class="col-sm-4 control-label">{@i18n key="Alias"/}* </label>
           <div class="col-sm-8">
             <input type="text" name="alias" class="form-control" placeholder="{@i18n key="Alias"/}" ng-model="profile.alias">
+            <div class="cd-tooltip">
+              {@i18n key="This  is shown to other users when your child uses the youth forums or if they choose to make their profile public. It should be a unique nickname. Try adding numbers at the beginning or end to make your alias unique."/}
```
Comment (sandrobonazzola) [inline web/public/templates/profiles/general-info.dust]: 2 spaces between This and is

**Comment by DanielBrierton** [comment]
This is being done as part of an overhaul of the add child page, set to be merged soon, so closing this PR.

**Pull request closed without merging**